src/app/containers/LinkedData/index.jsx

```diff
@@ -5,6 +5,11 @@ import { ServiceContext } from '#contexts/ServiceContext';
 import { RequestContext } from '#contexts/RequestContext';
 import getAboutTagsContent from './getAboutTagsContent';
 
+const noTrustProjectServices = ['scotland'];
```

Comment:
Would it be better if this is something that comes from the service configs, rather than having a hardcoded list here?

Reply:
@jimjohnsonrollings I think that is doable as it is, if we could instead do a check for ether of `publishingPrinciples` or `noBylinesPolicy` that way, we'd know if it's trusted or not as both values appear `null` for Scotland

Reply:
I think checking for `publishingPrinciples` would be a good proxy for this, yes.